cheatsheets/Secrets_Management_Cheat_Sheet.md

```diff
@@ -56,9 +56,28 @@ Rotating certain keys, such as encryption keys, might trigger full or partial da
 
 An additional level of security can be achieved by minimizing the time window
 where a secret is in memory and limiting the access to its memory space.
-However, this approach may be considered overkill in many cases, as it is only
-useful if the attacker already has access to the memory of the process handling
-the secret. By that point, the security breach may have already occurred.
+
+Depending on your particular circumstances, this can be difficult implement in a
+manner that ensures memory security. Since determining whether it is beneficial
```

Comment:
I think "Since" is the wrong word here. Is there a word that more clearly connects these two sentences?

Reply:
Would you like to me do a new PR for these edits? Sorry if I was going too fast.

Reply:
Sure